.github/init-environment

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -eu -o pipefail
+
+CACHE_ROOT=/var/cache/rchain-build
```

Comment:
Short description of how cache is working would be very helpful.

Reply:
Done.